internal/venice-common/src/main/java/com/linkedin/venice/utils/RegionUtils.java

```diff
@@ -44,4 +48,17 @@ public static String getRegionSpecificMetricPrefix(String localRegionName, Strin
       return localRegionName + "_from_" + regionName + "_remote";
     }
   }
+
+  /**
+   * A helper function to split a region list with {@link #REGION_FILTER_LIST_SEPARATOR}
+   */
+  public static Set<String> parseRegionsFilterList(String regionsFilterList) {
+    if (StringUtils.isEmpty(regionsFilterList)) {
+      return Collections.emptySet();
+    }
+    Set<String> regions = new HashSet<>();
+    String[] tokens = regionsFilterList.trim().toLowerCase().split(REGION_FILTER_LIST_SEPARATOR);
```

Comment:
I don't think we enforce lower case for region names. I might be wrong, though

Reply:
This method was originally in VeniceHelixAdmin and moved to this `venice-common` so I suppose it has been working this way.